utils/spacewalk-common-channels

```diff
@@ -106,6 +117,18 @@ def add_channels(channels, section, arch):
             channel[k] = config.get(section, k)
         channels[channel['label']] = channel
 
+
+def channel_exists(client, base_channel_label):
+    # check whether channel exists
+    try:
+        base_info = client.channel.software.getDetails(key, base_channel_label)
+        return base_info
+    # We should improve this as a connection failure would be the same
+    # as not finding the channel
+    except xmlrpclib.Fault, e:
```

Comment:
@mcalmer, is this acceptable? In the end it's the same we did before this change (when this was not a function the code continue in when there were any kind of exceptions.

Reply:
yes feel free to merge